dbt/models/iasworld/schema/iasworld.comdat.yml

```diff
@@ -101,21 +101,30 @@ sources:
           - name: chgrsn
             description: '{{ doc("column_chgrsn") }}'
             tests:
-              - expression_is_true:
-                  name: iasworld_comdat_chgrsn_eq_5_mktadj_not_null
-                  expression: |
-                    != '5' OR mktadj IS NOT NULL
+              - accepted_values:
```

Comment:
While the `expression_is_true` version of this test had a nice conciseness, I've always found it confusing to wrap my head around the double negative defined in the `expression`. In this PR, I've split each `expression_is_true` test out into three `accepted_values` tests, which are (in my opinion) more verbose but easier to understand.

Reply:
**praise:** It was definitely confusing, this is much more readable.

Reply:
**praise:** It was definitely confusing, this is much more readable.